app/graphs.py

```diff
@@ -42,4 +42,7 @@ def df_tech_stack_by_role(database: MongoDB) -> DataFrame:
     }, {"tech_stack": True}))
     mentors = mentors.explode(column="tech_stack")
     mentors["role"] = "Mentor"
-    return pandas.concat([mentees, mentors])
+    desc_para_final = "This graph shows the ratio of Mentors to Mentees. " \
```

Comment:
A great start to something that will evolve incrementally as we continue to work with Web on implementation. It's necessary that this come from the DS side, so we retain the ability to adjust the description as parameters and visualizations change.